### ISSUE TITLE
variables and counter

### ISSUE THREAD

**Issue by arjunashetty** [mzbench/mzbench#31]
hi team, 
How can we create counter variables and also how can we increment a number variable ?
i am looking to get a counter , which increments after each connection.
eg : numvar("n")

**Comment by ioolkos** [comment]
Here's the spec/docs of the Benchmark Definition Language (BDL): https://mzbench.github.io/mzbench/scenarios/spec/

**Comment by arjunashetty** [comment]
@ioolkos  
yes i referred to this link, i could not see anything w.r.t to incrementing a variable or creating a counter, can you please help how can we define.. ?

**Comment by arjunashetty** [comment]
@ioolkos  its not mentioned in the BDL doc mentioned above. I need to subscribe to multiple topics also publish on multiple topics, how can this be achieved..? kindly help.

**Comment by dhruvjain99** [comment]
@arjunashetty did you find any way to increment or create a counter variable?

**Comment by arjunashetty** [comment]
Hi Dhruv,

No I haven't got it.. :(.
